lib/active_record/connection_adapters/percona_adapter.rb

```diff
@@ -115,6 +115,10 @@ def remove_index(table_name, options = {})
       def error_number(_exception)
       end
 
+      def full_version
+        @mysql_adapter.raw_connection.server_info[:version]
```

Comment:
If would be great if we could use the `mysql_adapter` that we have already declared instead of directly accessing the ivar.